frontend/src/modules/search/components/SearchBox.js

```diff
@@ -322,16 +322,18 @@ export class SearchBoxBase extends React.Component<InternalProps, State> {
         );
 
         if (!selectedFilters.length) {
-            selectedFilters = [{ title: 'All' }];
+            selectedFilters = [{ name: 'All' }];
         }
 
         const selectedFiltersString = selectedFilters.map(
             item => {
+                // Specific case for Translation Authors filters
                 if (item.display_name) {
+                    // $FLOW_IGNORE: I just can't figure out the Flow error here.
```

Comment:
What is it?

Reply:
```
Cannot call this.props.authorsAndTimeRange.authors.filter because:
 • Either cannot get item.name because property name is missing in Author [1].
 • Or cannot add item.display_name and "'s translations" because property display_name of unknown type [2] could
   either behave like a string or like a number.

     src/modules/search/components/SearchBox.js
     310│         );
     311│
     312│         const authors = this.getSelectedAuthors();
     313│         const selectedAuthors = this.props.authorsAndTimeRange.authors.filter(
     314│             f => authors.includes(f.email)
     315│         );
     316│
     317│         let selectedFilters = [].concat(
     318│             selectedStatuses,
        :
 [2] 331│                 if (item.display_name) {

     src/modules/search/reducer.js
 [1]  13│     +authors: Array<Author>,

```

Reply:
OK, that makes sense to me. Basically here `item` can be either of type `Author` or type X (not sure where that one is defined, but it doesn't matter). Author has `display_name` and not `name`, X has `name` and not `display_name`. The problem here is that Flow is not able to tell, in each branch of the condition, which type of item is being used. Which makes, in its eyes, the instruction below invalid, because there are branches where `item.display_name` is not defined and thus could be anything.

I can't dive into this right now, but it might take roots in the definition of type "X". I think this definitely a solvable issue and we should not be ignoring it.

Reply:
Thanks! I'll try to change the logic a little bit.